<ui version="4.0" >
 <class>CallToaster</class>
 <widget class="QWidget" name="CallToaster" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <widget class="QFrame" name="windowFrame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>0</number>
      </property>
      <item row="2" column="0" >
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>6</number>
        </property>
        <property name="spacing" >
         <number>0</number>
        </property>
        <item row="0" column="0" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="3" >
         <widget class="RetroStyleLabel" name="pickUpButton" >
          <property name="frameShape" >
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="pixmap" >
           <pixmap resource="../images.qrc" >:/images/toaster/pickup.png</pixmap>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <widget class="RetroStyleLabel" name="hangUpButton" >
          <property name="frameShape" >
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="pixmap" >
           <pixmap resource="../images.qrc" >:/images/toaster/hangup.png</pixmap>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLabel" name="pixmapLabel" >
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="4" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="0" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>0</number>
        </property>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="closeButton" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>0</hsizetype>
            <vsizetype>0</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>18</width>
            <height>18</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>18</width>
            <height>18</height>
           </size>
          </property>
          <property name="icon" >
           <iconset resource="../images.qrc" >:/images/close-down.png</iconset>
          </property>
          <property name="autoRaise" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" >
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>0</number>
        </property>
        <item row="0" column="2" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1" >
         <widget class="QLabel" name="messageLabel" >
          <property name="palette" >
           <palette>
            <active>
             <colorrole role="WindowText" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Button" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>212</red>
                <green>208</green>
                <blue>200</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Light" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Midlight" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>234</red>
                <green>229</green>
                <blue>220</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Dark" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>128</red>
                <green>128</green>
                <blue>128</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Mid" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>141</red>
                <green>139</green>
                <blue>133</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Text" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="BrightText" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="ButtonText" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Base" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Window" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>212</red>
                <green>208</green>
                <blue>200</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Shadow" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>64</red>
                <green>64</green>
                <blue>64</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Highlight" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>10</red>
                <green>36</green>
                <blue>106</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="HighlightedText" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Link" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>0</red>
                <green>0</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="LinkVisited" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>0</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="AlternateBase" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>232</red>
                <green>232</green>
                <blue>232</blue>
               </color>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="WindowText" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Button" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>212</red>
                <green>208</green>
                <blue>200</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Light" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Midlight" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>234</red>
                <green>229</green>
                <blue>220</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Dark" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>128</red>
                <green>128</green>
                <blue>128</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Mid" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>141</red>
                <green>139</green>
                <blue>133</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Text" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="BrightText" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="ButtonText" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Base" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Window" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>212</red>
                <green>208</green>
                <blue>200</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Shadow" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>64</red>
                <green>64</green>
                <blue>64</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Highlight" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>212</red>
                <green>208</green>
                <blue>200</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="HighlightedText" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Link" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>0</red>
                <green>0</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="LinkVisited" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>0</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="AlternateBase" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>232</red>
                <green>232</green>
                <blue>232</blue>
               </color>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="WindowText" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>106</red>
                <green>104</green>
                <blue>100</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Button" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>212</red>
                <green>208</green>
                <blue>200</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Light" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Midlight" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>233</red>
                <green>231</green>
                <blue>227</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Dark" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>128</red>
                <green>128</green>
                <blue>128</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Mid" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>141</red>
                <green>139</green>
                <blue>133</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Text" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>106</red>
                <green>104</green>
                <blue>100</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="BrightText" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="ButtonText" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>106</red>
                <green>104</green>
                <blue>100</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Base" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>212</red>
                <green>208</green>
                <blue>200</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Window" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>212</red>
                <green>208</green>
                <blue>200</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Shadow" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Highlight" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>10</red>
                <green>36</green>
                <blue>106</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="HighlightedText" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Link" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>0</red>
                <green>0</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="LinkVisited" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>0</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="AlternateBase" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>232</red>
                <green>232</green>
                <blue>232</blue>
               </color>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
          <property name="text" >
           <string>Message</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RetroStyleLabel</class>
   <extends>QLabel</extends>
   <header>gui/toaster/RetroStyleLabelProxy.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images.qrc" />
 </resources>
 <connections/>
</ui>
